### PR TITLE
feat: add tier0_vrfs and tier0_vrf datasource

### PR DESCRIPTION
**Pull request by gaetanars**
Add first Data Source tier-0 VRF for this provider.

### CODE REVIEW

File: internal/cloudavenue/provider.go [hunk 1]
```diff
@@ -59,6 +62,12 @@ func (p *cloudavenueProvider) Schema(_ context.Context, _ provider.SchemaRequest
 			"url": schema.StringAttribute{
 				MarkdownDescription: "The URL of the CloudAvenue API. Can also be set with the `CLOUDAVENUE_URL` environment variable.",
 				Optional:            true,
+				Validators: []validator.String{
+					stringvalidator.RegexMatches(
+						regexp.MustCompile(`[a-z]$`),
```
Comment (azrod) [inline internal/cloudavenue/provider.go:67]: [This regex](https://regex101.com/r/jltDM2/1) not working for URL format and not checking if string is URL format.

Try this regex ([Regex101](https://regex101.com/r/jG5VyL/1))
`^(https|http):\/\/([\da-z\.-]+)\.([a-z\.]{2,6})(:[\d]{2,5})?([\/\w \.-]*)[a-z0-9]+$`

Comment (eremid) [reply internal/cloudavenue/provider.go:67]: Or just `^https?:\/\/\S+\w$` ?
https://regex101.com/r/QaGBU3/1

Comment (gaetanars) [reply internal/cloudavenue/provider.go]: Corrected

**Review by azrod — APPROVED** [review]

**Pull request merged**